packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx

```diff
@@ -168,7 +169,9 @@ export const TransactionReviewModalContent = ({
                 setIsSending={() => setIsSending(true)}
                 ethereumStakeType={ethereumStakeType || undefined}
             />
-            <TransactionReviewEvmExplanation account={selectedAccount.account} />
+            {!ethereumStakeType && (
+                <TransactionReviewEvmExplanation account={selectedAccount.account} />
```

Comment:
Could you do this check inside TransactionReviewEvmExplanation component please?

Reply:
sure, https://github.com/trezor/trezor-suite/pull/13732/commits/b3e350b47f6b661b602563fd6399b016e269e24f